src/compas_plotters/meshplotter.py

```diff
@@ -20,13 +20,26 @@
 
 
 def valuedict(keys, value, default):
-    value = value or default
-    if isinstance(value, dict):
-        valuedict = {key: default for key in keys}
-        valuedict.update(value)
-    else:
-        valuedict = {key: value for key in keys}
-    return valuedict
+    """
+     Build value dictionary from a list of keys and a value.
+
+     Parameters
+     ----------
+     keys: list
+         The list of keys
+     value: {dict, None}
+         A value or the already formed dictionary
+     default:
+         A default value to set if no value
+
+
+     Returns
+     -------
+     dict
+         A dictionary
+     """
+    value = value or dict.fromkeys(keys, default)
+    return {key: value.get(key, default) for key in keys}
```

Comment:
considering the fact that the `value` can also be a single value (see below), i am not sure this will produce the same outcome...

Reply:
Ok, the first test made me thing that value was either a dict or None. 
So, basically, if you use a single value, then you set this single value to the whole dict?

Reply:
yes that sounds about right